src/apps/opening-hours-editor/ConfirmAddRepeatedOpeningHour.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useText } from "../../core/utils/text";
+import { OpeningHoursCategoriesType } from "./types";
+
+type ConfirmAddRepeatedOpeningHourType = {
+  startDateString: string;
+  weekDayName: string;
```

Comment:
Please consider only passing the startdate as a prop - and preferably as a proper date.

When we have the start date as a prop we can determine the week day name from this:

1. We already have a function for this
2. It should not be expensive to do
3. We avoid bugs where the date and weekday may come out of sync